R/export_cdisc.R

```diff
@@ -201,6 +205,8 @@ export_cdisc <- function(res_nca) {
 
   # Include subject metadata and select adpp columns
   adpp <- pp_info %>%
+    # Elude potential collapse cases with PC variables
+    .[!names(.) %in% c("AVAL", "AVALC", "AVALU")] %>%
```

Comment:
issue: Try not mixing base R and dplyr patters, as the code becomes harder to comprehend. I would also clean up older code and do:
```R
  adpp <- pp_info %>%
    # Elude potential collapse cases with PC variables
    select(-any_of(c("AVAL", "AVALC", "AVALU"))) %>%
    rename(AVAL = PPSTRESN, AVALC = PPSTRESC, AVALU = PPSTRESU)  %>%
    left_join(
      res_nca$data$dose$data %>%
        select(-any_of(setdiff(names(pp_info), "USUBJID"))),
      by = "USUBJID"
    ) %>%
    select(any_of(adpp_col))
```

Reply:
Yes, I completely agree. I sometimes miss these things, I promise to pay more attention to this from now on!